evap/staff/templates/staff_semester_import.html

```diff
@@ -16,7 +16,11 @@ <h3>{% trans 'Import semester data' %}</h3>
     <form id="semester-import-form" method="POST" enctype="multipart/form-data" class="form-horizontal">
         <div class="card mb-3">
             <div class="card-body">
-                <p>{% trans 'Upload Excel file (' %}<a href="{% url 'staff:download_sample_xls' 'sample.xls' %}">{% trans 'Sample File' %}</a>{% trans '). This will create all containing students, teachers and courses and connect them. It will also set the entered values as default for all evaluations.' %}</p>
+                <p>{% trans 'Upload Excel file (' %}<a href="{% url 'staff:download_sample_xls' 'sample.xls' %}">{% trans 'Sample File' %}</a>,
+                
+                <a href="#" onClick="copyHeaders(['Degree', 'Student last name', 'Student first name', 'Student username', 'Student email address', 'Course kind', 'Course is graded', 'Course name (de)', 'Course name (en)', 'Responsible title', 'Responsible last name', 'Responsible first name', 'Responsible username', 'Responsible email address'])">
```

Comment:
We just merged a PR by @janno42 that removed the user name columns here (Student username and Responsible username).